src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/entity/CommunityOffenderManager.kt

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.entity
+
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.ManyToMany
+import javax.persistence.OneToMany
+import javax.persistence.Table
+import javax.validation.constraints.NotNull
+
+@Entity
+@Table(name = "community_offender_manager")
+data class CommunityOffenderManager(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @NotNull
+  val id: Long = -1,
+
+  @Column(unique = true)
+  val staffIdentifier: Long,
+
+  @Column(unique = true)
+  val username: String,
+
+  val email: String?,
+
+  @OneToMany(mappedBy = "responsibleCom")
```

Comment:
No firstname, lastname for ease of access without API calls?

Reply:
The idea for now was that we don't want to become a source of truth for this information ( + telephone number). It would be easy to have differing information if we fail to keep up to date with delius. We need staffId, username and email (for notify) within our system.. Everything else we just get from delius.